src/style.css

```
@@ -0,0 +1,110 @@
+body {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  margin: 0;
+  padding: 30px 14px 14px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 5px 45px;
+}
+
+.product__description {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin: 0 0 5px;
+}
+
+.product__code {
+  font-size: 11px;
+  line-height: 13px;
+  margin: 0 0 15px;
```

```suggestion
  margin-bottom: 15px;
```
looks prettier. IMHO